src/chapters/11-route-params.md

```diff
@@ -125,9 +151,11 @@ visit http://localhost:4200/tests?nocontainer&deterministic
 wait  #qunit-banner.qunit-pass
 ```
 
+## Accessing Parameters from Dynamic Segments
+
 Awesome! We're making such great progress.
 
-Ok, now that we have our `rental` route, let's finish up our `rental` page. The first step to doing this is making our route actually _do_ something. We added the route, but we haven't actually implemented it. So let's do that first by creating the route file.
+OK, now that we have our `rental` route, let's finish up our `rental` page. The first step to doing this is making our route actually _do_ something. We added the route, but we haven't actually implemented it. So let's do that first by creating the route file.
```

Comment:
I believe we use `Ok` throughout the tutorial (we should keep it consistent). That being said, I think we should just cut that filler word 🤷‍♀ Probably better to just start with "Now that we have..."

Reply:
Took out OK. I've never seen it written 'Ok', but I searched the entire tutorial and it uses 'Okay' which I'm OK with.

Reply:
Ok. Sounds good to me!